README.md

```diff
@@ -409,7 +409,7 @@ You can define multiple git platforms and define which platform to use for each
 ```
 platforms:
   - name: [OPTIONAL] Human readable name of the platform (eg: GitHub enterprise)
-  - id: Unique identifier of the platform. Used to map a platform to a project. Used in dependencies section
+  - id: Unique identifier of the platform. Used to map a platform to a project. Used in dependencies section. The ids: default-github and default-gitlab are reserved and are configured to use the default values.
```

Comment:
```suggestion
  - id: Unique identifier of the platform. Used to map a platform to a project. Used in dependencies section. The ids: github-public and gitlab-public are reserved and are configured to use the default values.
```

Reply:
If I correctly understood we should update these too ,right?

Reply:
Apart from this minor, it looks great to me

Reply:
Oh yes missed this, thanks!